packs/osquery.yml

```diff
@@ -0,0 +1,20 @@
+AnalysisType: pack
+PackID: PantherManaged.OSQuery
+Description: Group of all OSQuery detections
+PackDefinition:
+  IDs:
+    - Osquery.Linux.AWSCommandExecuted
+    - Osquery.Linux.LoginFromNonOffice
```

Comment:
Wondering how will we deal with "configuration required" detections? We want to manage them via packs, but should they be enabled with the non-configuration required detections?

Reply:
Huh, that's a good point -- had not considered the case for `configuration required`.

It'd be slick if we had a thing in the UI that specifically alerts users to detections requiring configuration

Reply:
It's a good question, typically we disabled by default detections that had the `configuration required` tag, but that doesn't make as much sense in the packs world. I think for now we should just include them,  and customers will need to know to search for that tag. In  the future we might have some CTAs in the UI  for these.